js/index.js

```diff
@@ -1,7 +1,38 @@
+/** @type {HTMLCanvasElement} */
+
+const canvas = document.getElementById('canvas');
+
+const ctx = canvas.getContext('2d');
+
+/* const roadImg = new Image ();
+roadImg.src = "/images/road.png"; */
+
+
+// Create the player - car
+const car = new Component(220, 500, 60, 120, "red", ctx);
```

Comment:
Here it would be something like this:
`const car = new Player(218, 550, 60, 100, ctx)`